astropy/cosmology/core.py

```diff
@@ -1789,12 +1789,12 @@ def phi_z(Om0, Ok0, kappa, y1, A, z):
 
             phi_z1 = phi_z(self._Om0, self._Ok0, kappa, y1, A, z1)
             phi_z2 = phi_z(self._Om0, self._Ok0, kappa, y1, A, z2)
-        # Get lower-right 0<b<2 solution in Om, Ol plane.
+        # Get lower-right 0<b<2 solution in Om0, Ode0 plane.
         # Fot the upper-left 0<b<2 solution the Big Bang didn't happen.
-        elif (0 < b) and (b < 2) and self._Om0 > self.Ol0:
-            def phi_z(Om0, Ok0, kappa, y1, A, z):
+        elif (0 < b) and (b < 2) and self._Om0 > self._Ode0:
+            def phi_z(Om0, Ok0, y1, y2, z):
                 return np.arcsin(np.sqrt((y1 - y2) /
-                                         (1 + z) * Om0 / abs(Ok0) + y1))
+                                         ((1. + z) * Om0 / abs(Ok0) + y1)))
```

Comment:
We're only supporting Python 3 now.  This can be left as `1+z`.

Reply:
Sounds good, I changed it back.